inc/deploygroup.class.php

```diff
@@ -819,4 +819,9 @@ public function showForComputer($computers_id)
         }
         echo "</table>";
     }
+
+
+    public static function getIcon() {
```

Comment:
Any idea what the result is, a screenshot would have been welcome (we cannot take time to test every small change proposal).
Also, the method breaks the CI.

Reply:
Updated this PR to include screenshot.